test/basic.js

```diff
@@ -17,4 +17,10 @@ describe('basic', function() {
       ]
     });
   });
+
+  // Not sure if we can will be able to run this test
+  it.skip('Should support Svelte 3 features', async function() {
```

Comment:
We do need to figure out how to test this, perhaps write a util to change installed svelte version or something, not sure how this is gonna work out

Reply:
I would suggest a couple of node modules with their own `package.json` inside the test dir maybe. worth an experiment!

Reply:
Sounds like a good idea, I'll experiment a lil with that idea. If it fails I guess we'll have to port all tests to Svelte 3 and drop tests for Svelte 2

Reply:
Found some hacky way to implement this using a monorepo and some weird commands